tests/app/views/test_application.py

```diff
@@ -20,3 +24,125 @@ def test_should_use_local_cookie_page_on_cookie_message(self):
             '<p>GOV.UK uses cookies to make the site simpler. <a href="/cookies">Find out more about cookies</a></p>'
             in res.get_data(as_text=True)
         )
+
+
+class TestHomepageSidebarMessage(BaseApplicationTest):
+    def setup(self):
+        super(TestHomepageSidebarMessage, self).setup()
+
+    @staticmethod
+    def _find_frameworks(framework_slugs_and_statuses):
+
+        _frameworks = []
+
+        for index, framework_slug_and_status in enumerate(framework_slugs_and_statuses):
+            framework_slug, framework_status = framework_slug_and_status
+            _frameworks.append({
+                'framework': 'framework',
+                'slug': framework_slug,
+                'id': index + 1,
+                'status': framework_status,
+                'name': 'Framework'
+            })
+
+        return {
+            'frameworks': _frameworks
+        }
+
+    @staticmethod
+    def _assert_message_container_is_empty(response_data):
+        empty_message_container = '<div class="framework-message column-one-third"></div>'
+        assert_in(
+            BaseApplicationTest._strip_whitespace(empty_message_container),
```

Comment:
Yeah, `self` works.  

Incidentally, the [buyer](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/tests/helpers.py#L72), [supplier](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/dcfd39653357aca908492f4fe93cabd6065c333d/tests/app/helpers.py#L77), and [admin](https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/614cc0655b100d30dff11b80e512baf23799c650/tests/app/helpers.py#L98) frontend apps all have their own versions of whitespace stripping methods that are all named slightly different things... :/